pkg/helper/client.go

```diff
@@ -19,11 +19,10 @@ import (
 
 const nmstateCommand = "nmstatectl"
 
-var interfacesFilter string
-
-func init() {
-	interfacesFilter = os.Getenv("INTERFACES_FILTER")
-}
+var (
+	globCompiled      glob.Glob
```

Comment:
Please give it a better name describing its use, not only what it is. interfacesFilterGlob maybe? Up to you.